### PR TITLE
Add Migration Docs

### PR DESCRIPTION
**Pull request by isharailanga**
This PR is to add the migration steps to the doc space. 

- Duplicated the https://uk.ob.docs.wso2.com/en/latest/install-and-setup/upgrading-the-solution/upgrading-the-solution/ content.
- Removed DCR references.
- Changed OB Migration Tool Command to `sh wso2server.sh -DobMigrationSpec=BG`
- Remove Reporting references.

### CODE REVIEW

File: en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+               onFail: function (context) {
+                   publishAuthData(context, "AuthenticationFailed", {'psuChannel': psuChannel});
+                   OTPFlow(context);
+               }
+           });
+       };
+       ```
+
+2. Update the common auth script of the existing service provider.
+     1. Go to the Identity Server Management Console `https://<IS_HOST>:9443/carbon` and log in as the admin user. ![service_providers](../../assets/img/install-and-setup/upgrading-the-solution/serivice-providers.png)
```
Comment (malshaniS) [inline en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md]: port should be 9446

Comment (isharailanga) [reply en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md]: Fixed in 9229e3ee378aef05dbedb5a051baf6973bf77693.

File: en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                   doLogin(context);
+               }
+           });
+       };
+        
+       var OTPFlow = function(context) {
+           executeStep(2, {
+                    //OTP-authentication
+               onSuccess: function (context) {
+                   context.selectedAcr = "urn:cds.au:cdr:2";
```
Comment (Akila94) [inline en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md]: This seems to be a CDS-specific common-auth script. Shall we add this generic common-auth script instead?
https://drive.google.com/file/d/11Fz8wypQZqnOUJMBkrb_sRoeYHMO7mjc/view?usp=sharing

File: en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                    doLogin(context);
+                }
+            });
+        };
+        
+        var OTPFlow = function(context) {
+            executeStep(2, {
+                     //OTP-authentication
+                onSuccess: function (context) {
+                    context.selectedAcr = "urn:cds.au:cdr:2";
```
Comment (Akila94) [inline en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md]: Same as before, let's update this script with a generic script.

File: en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md (hunk 2 above)
Comment (isharailanga) [reply en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md]: Fixed in 5ab485a6f260c6a14c23e5001d46d98e02fb8e69

File: en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md (hunk 3 above)
Comment (isharailanga) [reply en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md]: Fixed in 5ab485a6f260c6a14c23e5001d46d98e02fb8e69

**Review by Akila94 — APPROVED** [review]

**Pull request merged**